src/core/conversion/viewUtils.test.ts

```diff
@@ -296,6 +297,247 @@ describe('series metadata', () => {
             expect(series1.length).toEqual(1);
             expect(series1[0].precision).toEqual(contentVariable.values[1].contentComponent?.numberOfDecimals);
         });
+
+        it('returns the correct series without selectable variables', () => {
```

Comment:
I would suggest adding some test cases where the order of the variables and the variable values are changed. Also maybe create a helper function for building the test data to make the test functions bit more compact and easier to read.